src/main/java/internet/shop/Demo.java

```diff
@@ -1,121 +1,152 @@
 
         System.out.println(manufacturerService.get(3L));
-        System.out.println(manufacturerService.get(4L));
+        System.out.println(manufacturerService.get(2L));
+        System.out.println(manufacturerService.get(1L));
 
-        System.out.println(manufacturerService.delete(1L));
-        System.out.println(manufacturerService.delete(2L));
-        System.out.println(manufacturerService.delete(700L));
+        System.out.println(manufacturerService.getAll());
 
-        Manufacturer manufacturer3 = new Manufacturer();
-        manufacturer3.setCountry("Italian");
-        manufacturer3.setName("Pastarozo");
-        manufacturer3.setId(3L);
+        Manufacturer manufacturer4 = new Manufacturer();
+        manufacturer4.setId(2L);
+        manufacturer4.setCountry("India");
+        manufacturer4.setName("Elephant");
 
-        manufacturerService.update(manufacturer3);
-        System.out.println(manufacturerService.get(3L));
-        System.out.println(manufacturerService.get(100L));
-        /*System.out.println("====================================");
-        Driver driver1 = new Driver();
-        driver1.setId(1L);
-        driver1.setName("Vasya");
-        driver1.setLicenceNumber("123-987-312");
+        Manufacturer manufacturer5 = new Manufacturer();
+        manufacturer5.setId(1L);
+        manufacturer5.setCountry("China");
+        manufacturer5.setName("Samurai");
 
-        Driver driver2 = new Driver();
-        driver2.setId(2L);
-        driver2.setName("Petya");
-        driver2.setLicenceNumber("1023-61253");
+        manufacturerService.update(manufacturer4);
+        manufacturerService.update(manufacturer5);
 
-        Driver driver3 = new Driver();
-        driver3.setId(3L);
-        driver3.setName("Anton");
-        driver3.setLicenceNumber("1244-1235-1235664");
+        System.out.println(manufacturerService.get(2L));
+        System.out.println(manufacturerService.get(1L));
+
+        /*manufacturerService.delete(1L);
+        manufacturerService.delete(2L);
+        manufacturerService.delete(3L);*/
+        System.out.println(manufacturerService.getAll());
+        System.out.println("====================================");
 
         Car car1 = new Car();
-        car1.setDrivers(driver1);
         car1.setId(1L);
-        car1.setManufacturer(manufacturer);
         car1.setModel("Audi");
+        car1.setDrivers(driver1);
+        car1.setDrivers(driver2);
+        car1.setDrivers(driver3);
+        car1.setManufacturer(manufacturer1);
 
         Car car2 = new Car();
-        car2.setDrivers(driver1);
-        car2.setDrivers(driver2);
         car2.setId(2L);
+        car2.setModel("Tesla");
+        car2.setDrivers(driver1);
+        car2.setDrivers(driver3);
         car2.setManufacturer(manufacturer2);
-        car2.setModel("BMW");
 
         Car car3 = new Car();
+        car3.setId(2L);
+        car3.setModel("BMW");
         car3.setDrivers(driver1);
```

Comment:
use carService.addDriverToCar() method instead after you already called `carService.create(car3)`

Reply:
Done